pages/test_analytics/test_suites.md

```diff
@@ -35,6 +35,26 @@ Alternatively, you can create [scheduled builds](/docs/pipelines/scheduled-build
 
 Test Analytics reviews the test results to detect flaky tests after every test run.
 
+## Run issues
+
+>🛠 Currently in private beta
+> This feature is currently in private beta. If you would like to request access to Run issues, [contact us here](support@buildite.com)
+
+<%= image "run-issues.png", alt: "Screenshot of a run with issues displaying in a list, including flaky, slow and failures." %>
+
+Test Analytics will automatically detect issues for each run. For each test, we currently detect up to three issues: flaky, slow and failure, listed in the following order of urgency:
+
+- **Flaky:** [See section on detecting flaky tests](#detecting-flaky-tests).
+
+- **Slow:** Slowness is measured by the comparative performance of tests within the current run. The system automatically flags slow tests when the slowest 1% of tests take more than 15% of the overall run time. This overall percentage can be manually amended within the suite settings.
+
```

Comment:
@MelissaKaulfuss @KatieWright26 new sentence has been added to the end of this to inform users that the overall percentage can be amended within settings

Reply:
LGTM!